### DISCUSSION TITLE
Establishment of Working Groups

### DISCUSSION THREAD

**Discussion by maxammann** [maplibre/maplibre-rs#41] [Announcements]
maplibre-rs is still quite young but already has some modules which require a deeper insight into topics.
For example maplibre-rs will need experts in:

* Rendering
* Font Rendering (@FabianWildgrube)
* Style Evaluation
* iOS/macOS, Linux/Windows, Android and Web (WebAssembly)
* etc.

I think it could make sense to have "domain experts" for each of these areas. It's easily possible to parallelize work on the above topics.

Anyone interested in owning one of the above topics? :)

**Comment by DerKarlos** [comment]
May be I could do some iOS/MacOS testing
